dev/bootstrap.sh

```
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

需要添加阻断参数：
set -u
set -e
set -o pipefail
避免错误被放大

---

@AdmondGuo set -u 我觉得有必要。因为，用户直接执行 bootstrap.sh ，BYZER_HOME 环境变量未定义时，会出现问题。直接报错 Fail Fast 是个好的方式。